exporter/util.go

```diff
@@ -1166,3 +1168,40 @@ func ListParallel(a workspace.NotebooksAPI, path string, shouldIncludeDir func(w
 	defer answer.MU.Unlock()
 	return answer.data, nil
 }
+
+var (
+	maxRetries        = 5
+	retryDelaySeconds = 2
+	retriableErrors   = []string{"context deadline exceeded", "Error handling request", "Timed out after "}
+)
+
+func isRetryableError(err string, i int) bool {
+	if i < (maxRetries - 1) {
+		for _, msg := range retriableErrors {
+			if strings.Contains(err, msg) {
+				return true
+			}
+		}
+	}
+	return false
+}
+
+func runWithRetries[ERR any](runFunc func() ERR, msg string) ERR {
```

Comment:
Rest looks good to me, only concern is that we can use https://github.com/databricks/databricks-sdk-go/blob/main/retries/retries.go  here otherwise, we would have to maintain this retry mechanism separately.

If the change required is urgent and using retries would require major refactoring, then we can go ahead with this. What do you think? @mgyucht

Reply:
`retries` package is a convenience, not a core part of the SDK. The retries package does not expose anything to e.g. print when a retry is taking place or print a specific message after including the total number of retries that have happened, so if @alexott needs these, it's perfectly fine for him to implement his own retry logic.

Reply:
Though @alexott curious if we added things like `WithMaxRetries` or `WithRetryOnErrorSubstrings` if that would satisfy your need.

Reply:
yep, let discuss if we can make something generic...